src/backend/distributed/commands/table.c

```diff
@@ -690,6 +900,47 @@ GetAlterTableCommandFKeyConstraintList(AlterTableCmd *command)
 }
 
 
+/*
+ * GetRangeVarListFromConstraintList returns a list of RangeVar objects for
+ * right relations in constraintList.
+ */
+static List *
+GetRangeVarListFromConstraintList(List *constraintList)
```

Comment:
This name looks too generic: `GetRangeVarListFromForeignConstraintList `

Reply:
2014e976980bb84d1bbe74218bb2ca7403ec9689